app/views/device_stories/_time_range_select.erb

```diff
@@ -0,0 +1,3 @@
+<%= form_tag controller_name ,method: :get, id: 'time_range_form' do %>
```

Comment:
@eitoball or @ma2gedev might be good folks to weigh-in here, but pretty sure you can write this as `form_with` on `@device_story` to get back to the same path.

Reply:
```suggestion
<%= form_with model: @device_story, method: :get, id: 'time_range_form' do %>
```

Reply:
Thank you! that worked like a charm.